src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -322,6 +350,24 @@ private PEModuleSymbol ContainingPEModule
             }
         }
 
+        public override RefKind RefKind
```

Comment:
>RefKind [](http://example.com/codeflow?start=32&length=7)

Do we have direct tests for the added APIs?

Reply:
Yes, `RefFieldTests` tests `RefKind` and `RefCustomModifiers` for various `FieldSymbol` implementations.